spec/inputs/http_spec.rb

```diff
@@ -544,6 +544,24 @@ def setup_server_client(url = self.url)
       expect { subject.register }.to_not raise_exception
     end
 
+    context "during run" do
+      let(:http_server) {
+        http_server = double(:http_server)
+        allow(http_server).to receive(:close)
+        allow(http_server).to receive(:run)
+        http_server
+      }
+      before(:each) do
+        allow(subject).to receive(:create_http_server).and_return(http_server)
+        subject.register
+      end
+      it "should show ssl is disabled" do
+        #[2024-10-18T10:09:33,199][INFO ][logstash.inputs.http     ][main][0d48] Starting http input listener {:address=>"0.0.0.0:3333", :ssl_enabled=>true}
```

Comment:
do you want to keep this line? If so, it should be :ssl_enabled => false

Reply:
I prefer to keep it, the incongruence has been fixed :+1: